package/src/test.rs

```diff
@@ -38,7 +40,7 @@ fn decode_output(output: Vec<u8>) -> Result<String> {
 }
 
 fn assert_stderr_output(command: &mut Command, expected_messages: Vec<&str>) -> Output {
-    let output = execute(command.stderr(Stdio::piped())).unwrap();
+    let output = execute_no_error(command.stderr(Stdio::piped()));
```

Comment:
I worry that this does no longer make the same assertion about the sub process' exit code as before for all the unrelated tests using this method now.

Reply:
As of #296, callers of `assert_stderr_output` can indicate whether the command should either succeed or fail.